src/snowflake/connector/connection.py

```diff
@@ -285,6 +285,10 @@ def _get_private_bytes_from_file(
         False,
         bool,
     ),  # log raw arrow chunk for debugging purpuse in case there is malformed arrow data
+    "disable_saml_url_check": (
+        False,
```

Comment:
minor suggestion to keep it `enable_saml_url_check` defaulting to `true` so we don't have a double negative

Reply:
good suggestion, but I found ODBC already named it disable_xxx in the PR: https://github.com/snowflakedb/snowflake-odbc/pull/557/files.
I feel it's better we align with them